frontend/src/modules/Organizations/views/OrganizationView.js

```diff
@@ -40,24 +42,7 @@ import {
 } from '../components';
 import { MetadataAttachment } from '../../Metadata_Forms/components';
 import { isModuleEnabled, ModuleNames } from 'utils';
-
-const tabs = [
-  { label: 'Overview', value: 'overview', icon: <Info fontSize="small" /> },
-  { label: 'Environments', value: 'environments', icon: <FaAws size={20} /> },
-  {
-    label: 'Metadata',
-    value: 'metadata',
-    icon: <FaAws size={20} />,
-    active: isModuleEnabled(ModuleNames.METADATA_FORMS)
-  },
-  {
-    label: 'Teams',
-    value: 'teams',
-    icon: <SupervisedUserCircleRounded fontSize="small" />
-  }
-];
-
-const activeTabs = tabs.filter((tab) => tab.active !== false);
+import { listRulesThatAffectEntity } from '../../Metadata_Forms/services';
```

Comment:
Alongside the whole PR and the implementation of the UI components of metadata forms there are violations of the modularization principles established in data.all v2.

The idea is that any of the FE modules can be replaced or removed without affecting other modules. Anything that is shared across modules is placed in the global packages. That means that imports such as `'../../Metadata_Forms/services';` that go 2 levels up and get an API from another module are "forbidden" because if we were now to remove or alter the Organizations module other modules would be affected. 

The whole MF module needs to be reviewed, but I think this task can be left out of this PR.